src/app.ts

```diff
@@ -554,6 +572,27 @@ function registerAPIRoutes(
   );
 }
 
+function registerBonusAPIRoutes(
+  app: Express,
+  basePath: string,
+  bonusService: BonusService
+): void {
+  const bearerSessionTokenAuth = passport.authenticate("bearer.session", {
```

Comment:
we must call `passport.authenticate("bearer.session", ..)` only once. in case, put this call outside this method (at top level) and remove the inner call everywhere it is